packages/ui-components/src/components/text-field/text-field.utils.ts

```diff
@@ -0,0 +1,19 @@
+import { COMMON_INPUT_MASK_CONFIG } from './text-field.config';
+import { EInputFieldType } from './text-field.types';
+
+export function getInputMaskConfig(type: EInputFieldType): Inputmask.Options {
+	if (type === EInputFieldType.Number) {
```

Comment:
can you invert the logic here? return `{ ...COMMON_INPUT_MASK_CONFIG }` immediately if `type !== EInputFieldType.Number`?

Reply:
Sure :)